test/fixtures/kitchen/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "google-auth-library-nodejs-fixture",
+  "version": "1.0.0",
+  "description": "An app we're using to test the library. ",
+  "scripts": {
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check",
+    "start": "node build/src/index.js"
+  },
+  "keywords": [],
+  "author": "Justin Beckwith <justin.beckwith@gmail.com>",
+  "license": "MIT",
```

Comment:
Use Apache 2.0, same as the parent project.

Reply:
done